snap/snapenv/snapenv.go

```diff
@@ -100,6 +102,47 @@ func componentEnv(info *snap.Info, component *snap.ComponentInfo) osutil.Environ
 // used by so many other modules, we run into circular dependencies if it's
 // somewhere more reasonable like the snappy module.
 func basicEnv(info *snap.Info) osutil.Environment {
```

Comment:
BasicEnv was quite simple before, I think this change should be made a helper function so we keep the basicEnv function a bit clean, I think it makes the function too complex.

Reply:
Moved to helper